harness/determined/experimental/client.py

```diff
@@ -289,10 +289,13 @@ def logout() -> None:
 
 
 @_require_singleton
-def list_users() -> List[User]:
+def list_users(active_only: bool) -> List[User]:
```

Comment:
this should be the same exact interface as the class method (but singleton)
```suggestion
def list_users(active: Optional[bool] = None) -> List[User]:
```